framework/test/unit/streaming/kafka-api.test.ts

```diff
@@ -1250,4 +1250,64 @@ describe('Using global removal policy and DELETE construct removal policy ', ()
       DeletionPolicy: 'Delete',
     });
   });
+});
```

Comment:
can you separate this one please into its own e2e or make use of this one kafka-api-byo-service-token.e2e.test.ts